blueocean-rest-impl/src/main/java/io/jenkins/blueocean/service/embedded/rest/QueueItemImpl.java

```diff
@@ -59,7 +59,7 @@ public String getOrganization() {
 
     @Override
     public String getPipeline() {
-        return pipeline.getName();
+        return pipeline == null ? "" : pipeline.getName();
```

Comment:
When will `pipeline` be null? Should it not be an error? Not sure if returning empty string is going to eventually cause confusing error otherwise?

Reply:
Well not sure exactly when I got this. Probably because of wrong usage on my side of QueueItemImpl.